File: p001.hs
{- 
 - Solution to Project Euler problem 1
 - by Project Nayuki
 - 
 - http://www.nayuki.io/page/project-euler-solutions
 - https://github.com/nayuki/Project-Euler-solutions
 -}


main = putStrLn (show ans)
ans = sum [x | x <- [0..999], (mod x 3) == 0 || (mod x 5) == 0]

File: p001.py
# 
# Solution to Project Euler problem 1
# by Project Nayuki
# 
# http://www.nayuki.io/page/project-euler-solutions
# https://github.com/nayuki/Project-Euler-solutions
# 


ans = sum([x for x in range(1000) if (x % 3 == 0 or x % 5 == 0)])
print(ans)
